DummyServer/Controllers/BranchController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using BuildNotifications.Plugin.DummyServer;
+using DummyServer.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DummyServer.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class BranchController
```

Comment:
There is a `ControllerBase` class in Asp.NET Core with some neat helper methods.

A controller method would always return an `IActionResult` or `IActionResult<T>` and use helper methods like `return Ok( result );` or `return NotFound();`

Reply:
I'm familiar with this. However, since the request answered just fine, I figured ASP.NET would automatically Ok(), if the return type is void. Guess it's bad practice